sdk/tests/js/tests/tex-input-validation.js

```diff
@@ -310,7 +310,7 @@ var testCases = [
     border: 0,
     format: gl.RGBA,
     type: gl.BYTE,
-    expectedError: gl.INVALID_ENUM },
+    expectedError: gl.INVALID_OPERATION},
```

Comment:
I think INVALID_ENUM is correct for ES2, where BYTE is not a valid enum for type.